src/Helpers.re

```diff
@@ -9,42 +9,42 @@ type transaction = {
 	[@bs.as "type"] type_ : string
 };
 
-[@bs.deriving abstract]
+
 type card_address_geo = {
 	lat : string,
 	lng : string
 };
 
-[@bs.deriving abstract]
+
 type card_company = {
 	name : string,
 	catchPhrase : string,
 	bs : string
 };
 
-[@bs.deriving abstract]
+
 type card_post = {
 	words : string,
 	sentence : string,
 	sentences : string,
 	paragraph : string
 };
 
-[@bs.deriving abstract]
+
 type card_address = {
-	[@bs.optional] street : string,
-	[@bs.optional] streetA : string,
-	[@bs.optional] streetB : string,
-	[@bs.optional] streetC : string,
-	[@bs.optional] streetD : string,
+	street : option(string),
```

Comment:
I think doing this won’t handle null and undefined, just undefined. So BuckleScript will just look for undefined before calling it and might potentially crash

Reply:
So change it to `Js.Nullable.t(string)`?